pandora-common/src/assets/appearanceActions.ts

```diff
@@ -945,13 +945,25 @@ export function ActionRoomDeviceDeploy(processingContext: AppearanceActionProces
 	if (!manipulator.modifyItem(itemId, (it) => {
 		if (!it.isType('roomDevice'))
 			return null;
+
+		if (deployment === null && it.slotOccupancy.size > 0) {
```

Comment:
`deployment` can never be `null`, this likely wants to check that it has `deployed: false`?

Reply:
yep and this should have fixed the issue Claudia mentioned above